package.json

```diff
@@ -2,7 +2,7 @@
   "name": "keysim",
   "version": "1.3.0",
   "description": "Keyboard simulator for JavaScript.",
-  "main": "dist/keysim.js",
+  "main": "lib/keysim.js",
   "jsnext:main": "lib/keysim.js",
```

Comment:
I read a bit in the rollup project and it seems to me that this is still correct.